backend/templates/create-api/index.js

```diff
@@ -34,7 +34,7 @@ const templateFilePaths = templateTypes.map((type) => {
  * @param {*} message The message to print.
  */
 const sayError = (message) => {
-  console.error(`${consoleErrorColor}${message}${consoleClearColors}`);
+  console.log(`${consoleErrorColor}${message}${consoleClearColors}`);
```

Comment:
why were these changed?

Reply:
Because console.success does not work in the cli.  The linter picked them up last time and did an auto fix which was converted them from console.log to console.success.